tests/lib/pkgdb.sh

```diff
@@ -559,14 +559,14 @@ pkg_dependencies_ubuntu_classic(){
 }
 
 pkg_linux_image_extra(){
-    available_pkgs=$(apt-cache policy linux-image-extra-.*-generic | grep -E 'linux-image-extra-.*:' | tr -d :)
-    if echo "$available_pkgs" | grep -q "linux-image-extra-$(uname -r)"; then
-        echo "linux-image-extra-$(uname -r)"
-        return
+    if apt-cache policy "linux-image-extra-$(uname -r)" >/dev/null 2>&1; then
+        echo "linuximage-extra-$(uname -r)"
```

Comment:
@sergiocazzolato there's a typo here in `echo "linuximage-extra-$(uname -r)"`, missing space, should be `echo "linux-image-extra-$(uname -r)"`

Reply:
@bboozzoo, I'll test that right now